### ISSUE TITLE
Version incomplete in SPEC file

### ISSUE THREAD

**Issue by rinigus** [mer-hybris/bluebinder#18]
Please set the full version in SPEC file as a part of the release. Right now we have 1.0 for 1.0.12

**Comment by monich** [comment]
I made it [1.0](https://github.com/mer-hybris/bluebinder/commit/6b71209) because it wasn't being kept in sync with the reality. Did you like 1.0.0 better?

1.0 at least isn't wrong, it's just partial. And it's actually ignored by obs builds (replaced with something derived from the latest tag), that's why (almost) nobody cares.

**Comment by rinigus** [comment]
Yes, I noticed. But for most of the packages on SFOS we have correct version information set as a part of the releasing process.

OBS does ignore it, indeed. However, community OBS is not up yet for working on ports with it. So, while building with tbuilder, I will get 1.0 as a version (+ release component). 

I personally think that we should not assume building on OBS or building with git tags. If the SPEC is provided, I expect it to be in sync as a part of the release process.

Note that we have few issues like this for several SFOS packages leading to the discussion on whether git tags are good enough. See example at https://github.com/mer-hybris/qt5-qpa-hwcomposer-plugin/issues/89

**Comment by rinigus** [comment]
Just an additional note: I don't expect it to be fixed for the current version. But maybe it would be possible to incorporate the version tracking in future as a part of the release process.

**Comment by monich** [comment]
Well, I'm keeping .spec version in sync with the latest tag for the packages that I maintain, so I'm definitely for it )) I wasn't considering myself a maintainer of this package, though.
